frontend/src/static/js/v_ramp.js

```diff
@@ -74,7 +85,13 @@ window.vRamp = {
       return new Date(untilDate) - new Date(sinceDate);
     },
     getSliceColor(date) {
-      const timeMs = (new Date(date)).getTime();
+      let timeMs;
+      if (window.hashParams.timeframe !== this.tframe) {
```

Comment:
can explain the need to add this logic?

Reply:
> Though zoomfromramp is removed, I still need a mean to distinguish the two for the getSliceColor function. As the color is calculated using date and we want same color for all commit ramps to look like the same week/day ramp, when the zoom is opened by clicking ramp with week granularity, we need to use sdate instead of date in the getSliceColor.

Ok noted. @0blivious this comment addresses this question instead right?

Reply:
ya, sorry under wrong section